common/persistence/persistence_metric_clients.go

```diff
@@ -1293,6 +1294,10 @@ func (p *metricEmitter) recordRequestMetrics(operation string, caller string, la
 func updateErrorMetric(handler metrics.Handler, logger log.Logger, operation string, err error) {
 	if err != nil {
 		metrics.PersistenceErrorWithType.With(handler).Record(1, metrics.ServiceErrorTypeTag(err))
+		if common.IsContextCanceledErr(err) {
```

Comment:
should we just put `*serviceerror.Canceled` and `context.Canceled` in the "no-op" branch of the switch?

that doesn't handle some wrapped errors but neither do the existing cases below

Reply:
`context.Canceled` is not a type. It is concrete error instance. Having 2 "canceled" errors separately seems worse.